src/components/lyne-teaser/styles/lyne-teaser.component.scss

```
@@ -0,0 +1,86 @@
+@mixin image-aspect-ratio-4-3 {
+  aspect-ratio: 4 / 3;
+}
+
+:host {
+  --hover-scale-102: 1.02;
```

We always try to prevent adding values into the name.
```suggestion
  --hover-scale: 1.02;
```